lib/patientFinder/findPatientsIDs.rb

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/ruby
+require 'rubygems'
+require 'nokogiri'
+require 'health-data-standards'
+require 'zip/zipfilesystem'
+require 'fileutils'
+require 'health-data-standards'
+
+#a simple patient from QRDA - contains the source files name and the parsed record associated
+class PatientFromQRDA
```

Comment:
In general, I think one class per file is a good idea, even if the classes are really short like this